packages/@angular/cli/tasks/serve.ts

```diff
@@ -182,7 +182,11 @@ export default Task.extend({
           return reject(err);
         }
         if (serveTaskOptions.open) {
-            opn(serverAddress);
+           if (serveTaskOptions.browser) {
+            opn(serverAddress, { app: serveTaskOptions.browser });
```

Comment:
@clydin @filipesilva should be cross platform https://github.com/sindresorhus/opn#api